melodic-manatees/early_internet/main/templates/main/main-about.html

```diff
@@ -1,5 +1,20 @@
 {% extends 'main/main-base.html' %}
 
 {% block content%}
-<h1>this is about stuff</h1>
+<div>
+    <h2>About the Project:</h2>
+    <section>
+        <h1>The project</h1>
+        <p>words go here about it</p>
+    </section>
+    <section>
+        <h1>The Team: Melodic-Manatees</h1>
+        <p>r00t ^blm (sadiq)#3476 - Lead</p>
+        <p>Demonkingj#4796</p>
+        <p>el_cocodrilo#7140</p>
+        <p>honorablebrah#5855</p>
```

Comment:
<p>EM#2961</p> is missing

Reply:
Very strange, it's showing on my branch locally.  Let me recommit!